lightning/src/ln/functional_test_utils.rs

```diff
@@ -530,23 +530,26 @@ macro_rules! get_event_msg {
 	}
 }
 
+/// Get an error message from the pending events queue.
+pub fn get_err_msg(node: &Node, recipient: &PublicKey) -> msgs::ErrorMessage {
```

Comment:
It looks like it would be easy to get rid of the macro entirely here, it's not called in that many places